### PR TITLE
chore: add new lab

### PR DESCRIPTION
**Pull request by hacktivist123**
This PR adds the new Migrating from Calico lab

### CODE REVIEW

File: content/labs/2024-02-12-migrating-from-calico/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+date: '2024-02-12T14:00:00.000Z'
+externalUrl: 'https://isovalent.com/labs/migrating-from-calico/'
+ogImage: cover.png
+categories:
+  - Networking
+place: Online
+title: 'Migrating From Calico'
+ogSummary: 'Migrating to Cilium from another CNI is a very common task. But how do we minimize the impact during the migration? How do we ensure pods on the legacy CNI can still communicate to Cilium-managed during pods during the migration? How do we execute the migration safely, while avoiding a overly complex approach or using a separate tool such as Multus? With the use of the new Cilium CRD CiliumNodeConfig, running clusters can be migrated on a node-by-node basis, without disrupting existing traffic or requiring a complete cluster outage or rebuild. In this lab, you will migrate your cluster from Calico to Cilium.'
```
Comment (xmulligan) [inline content/labs/2024-02-12-migrating-from-calico/index.md:9]:
```suggestion
ogSummary: 'Learn how to migrate your cluster from Calico to Cilium using the new Cilium CRD CiliumNodeConfig. It allows running clusters to be migrated on a node-by-node basis, without disrupting existing traffic or requiring a complete cluster outage or rebuild.'
```

Comment (hacktivist123) [reply content/labs/2024-02-12-migrating-from-calico/index.md:9]: `we've mostly used the actual lab description for this page, that's why I did not bother editing it. If we do this then we'd need to update the rest of the labs to make everything be uniform.

Comment (xmulligan) [reply content/labs/2024-02-12-migrating-from-calico/index.md:9]: I just thought it was a bit weird for two labs to have the same description right next to each other

Comment (xmulligan) [reply content/labs/2024-02-12-migrating-from-calico/index.md:9]: even though they are different labs. might be confusing to the user

Comment (hacktivist123) [reply content/labs/2024-02-12-migrating-from-calico/index.md:9]: Fair enough, lets do it

Comment (xmulligan) [reply content/labs/2024-02-12-migrating-from-calico/index.md]: Thanks, I think we can leave the rest for now

**Review by xmulligan — APPROVED** [review]

**Pull request merged**